daily_word_bot/utils.py

```diff
@@ -102,10 +102,18 @@ def build_available_commands_msg(bot_commands: List[BotCommand]) -> str:
 hero_char = "🦸🏿‍♀️"
 
 
+def convert_to_hyperlink(url: str, text: str) -> str:
+    return f"<a href=\"{url}\">{text}</a>"
```

Comment:
rename function to hyperlink. Why? to follow the "imperative function naming" convention